lms/templates/header/header.html

```diff
@@ -45,6 +46,21 @@
 % endif
 </%block>
 
+% if configuration_helpers.get_value('ENABLE_UNSUPPORTED_BROWSER_ALERT', settings.FEATURES.get('ENABLE_UNSUPPORTED_BROWSER_ALERT', False)):
+  <script>
+  var $buoop = {notify:${configuration_helpers.get_value('UNSUPPORTED_BROWSER_ALERT_VERSIONS', settings.FEATURES.get('UNSUPPORTED_BROWSER_ALERT_VERSIONS', "{i:-10,f:-3,o:-3,s:-3,c:-3}"))|h},
```

Comment:
Do we need to duplicate this value here, if it's set in [lms.env.common](https://github.com/edx/edx-platform/pull/17216/files#diff-82bda515879c96ab0641bfcbe58631c3R429)?

Reply:
True, gonna make the correction